test/terra/backends/qasm_simulator/qasm_snapshot.py

```diff
@@ -30,6 +30,10 @@
     snapshot_state_pre_measure_statevector_nondeterministic,
     snapshot_state_post_measure_statevector_nondeterministic)
 
+def get_snapshots(data, label, snapshot_type):
+        """Format snapshots as list of Numpy arrays"""
```

Comment:
I think this has one more level of indentation, right?

Reply:
indeed!

Reply:
Will merge here and fix in #380